index.html

```diff
@@ -1,33 +1,209 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
     <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 </head>
+
 <body>
     <header class="header">
-            <a href="#" class="logo">My Logo</a>
-            <nav>
-                <i class="material-symbols-outlined menu_icon">
-                    menu
-                </i>
-            </nav>
+        <a href="#" class="logo">My Logo</a>
+        <nav>
+            <i class="material-symbols-outlined menu_icon">
+                menu
+            </i>
+        </nav>
     </header>
     <section class="container">
         <h1>I’m Abdurahim Glad to see you!</h1>
-        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
         <h2 class="social_media">LET’S CONNECT</h2>
         <ul class="social_media_icons">
             <li><a href="#"><img src="images/icons/Twitter.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Media.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Github.svg" alt="twitter"></a></li>
-            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/icons/Linkedin icon.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/icons/Media.svg" alt="media"></a></li>
+            <li><a href="#"><img src="images/icons/Github.svg" alt="github"></a></li>
+            <li><a href="#"><img src="images/icons/Sideshow.svg" alt="slideshow"></a></li>
         </ul>
     </section>
+    <section class="works_container">
+        <div class="card">
+            <img src="images/Snapshoot Portfolio.png" alt="">
+            <div class="work_desc">
+                <h1>Tonic</h1>
```

Comment:
- [ ] You should have only one h1 element in your entire HTML document. Kindly change all other h1 elements to hx(that is between h2 to h5). This is good SEO practice.